server/api/chat/agents.ts

```
@@ -3714,8 +3714,12 @@ export const AgentMessageApi = async (c: Context) => {
     message = decodeURIComponent(message)
     rootSpan.setAttribute("message", message)
     let ids
+    let isValidPath: boolean = false
     if (path) {
       ids = await getRecordBypath(path, db)
+      if (ids != null) {
+        isValidPath = Boolean(true)
+      }
```

If path is given and we somehow can't extract ids, can we throw an Invalid Path error?